web/src/enterprise/campaigns/detail/CampaignDetails.tsx

```diff
@@ -67,21 +66,18 @@ export const CampaignDetails: React.FunctionComponent<CampaignDetailsProps> = ({
     queryChangesetCountsOverTime,
     deleteCampaign,
 }) => {
-    /** Retrigger fetching */
-    const campaignUpdates = useMemo(() => new Subject<void>(), [])
-
     useEffect(() => {
         telemetryService.logViewEvent(campaignID ? 'CampaignDetailsPage' : 'NewCampaignPage')
     }, [campaignID, telemetryService])
 
     const campaign: CampaignFields | null | undefined = useObservable(
         useMemo(
             () =>
-                merge(of(undefined), campaignUpdates).pipe(
-                    switchMap(() => fetchCampaignById(campaignID)),
+                fetchCampaignById(campaignID).pipe(
+                    repeatWhen(notifier => notifier.pipe(delay(5000))),
                     distinctUntilChanged((a, b) => isEqual(a, b))
```

Comment:
Meta-comment: I saw this line and wanted to change it to because that's equivalent, as far as I understand my first-class functions

```suggestion
                    distinctUntilChanged(isEqual)
```

But then I get an "Unsafe assignment of an any value" :( Is that the reason why it uses the wrapping function?

Reply:
Exactly 👍